docker-images/ziti-console/README.md

```diff
@@ -0,0 +1,37 @@
+
+# openziti/ziti-console
+
+## Overview
+
+This is the preferred image for including the Ziti Administration Console (ZAC) in a multi-stage build of the ziti-controller image. The controller has an optional API binding `zac` that creates a `/zac` handler on the `edge-management` API's web listener.
```

Comment:
> This is the preferred image

Makes me ask -- what are the other images? I think this should just be simiplifed... "This image is used by the ziti-controller docker image to embed a version of ZAC within the ziti controller image". All the other stuff is just fluff that's not relevant to this particular docker file. It's also more words that can be made obsolete someday. I'd really perfer to keep this .md very lean since really, nobody in the world will use this unless they want to embed the ZAC SPA into their docker container too. That's how the whole .md should be written. "Read this if you want to embed the ZAC into your own docker image..." which nobody will want to do and that should dissuade them from reading further.